tests/dualtor/test_orchagent_active_tor_downstream.py

```diff
@@ -160,29 +161,28 @@ def test_downstream_ecmp_nexthops(
     add_nexthop_routes(rand_selected_dut, dst_server_addr, nexthops=nexthop_servers)
 
     try:
-        logging.info("Verify traffic to this route destination is distributed to four server ports")
-        check_nexthops_balance(rand_selected_dut, ptfadapter, dst_server_addr, tbinfo,
-            nexthop_interfaces, nexthops_count)
+        logging.info("Verify traffic to this route destination is sent to single downlink or uplink")
+        check_nexthops_single_downlink(rand_selected_dut, ptfadapter, dst_server_addr,
+                                       tbinfo, nexthop_interfaces)
 
         ### Sequentially set four mux states to standby
+        nexthop_interfaces_copy = nexthop_interfaces.copy()
```

Comment:
Hi @Ndancejic, test failed on testbed because we are using python2 in 202205 and the list in python2 doesn't have the copy() method. Do you also run by python2 in 202205 branch? If yes, need to change the copy here.

Reply:
I removed the copy method and replaced it, please take a look and let me know if that works for you.